blog/2022-05-03-serviceinfo-model-improvements.md

```diff
@@ -0,0 +1,12 @@
+---
+author: epenet
+authorURL: https://github.com/epenet
+title: "ServiceInfo model improvements and deprecations"
+---
+
+As of Home Assistant Core 2022.6, access to discovery information via the, previously deprecated, dictionary methods have been removed.
+
+
+This applies to the `DhcpServiceInfo`, `MqttServiceInfo`, `SsdpServiceInfo`, `UsbServiceInfo`, and `ZeroconfServiceInfo` instances.

```

Comment:
Please link to the relevant dev docs page.

Reply:
Are these the best location to link?
https://developers.home-assistant.io/docs/creating_integration_manifest/#dhcp
https://developers.home-assistant.io/docs/creating_integration_manifest/#mqtt
https://developers.home-assistant.io/docs/creating_integration_manifest/#ssdp
https://developers.home-assistant.io/docs/creating_integration_manifest/#usb
https://developers.home-assistant.io/docs/creating_integration_manifest/#zeroconf

Reply:
Maybe we should add a paragraph with some text and example of a discovery step in the config flow docs?

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#discovery-steps